src/test/java/com/github/dockerjava/core/command/ExecCreateCmdImplTest.java

```diff
@@ -44,7 +44,7 @@ public void afterMethod(ITestResult result) {
     public void execCreateTest() {
         String containerName = "generated_" + new SecureRandom().nextInt();
 
-        CreateContainerResponse container = dockerClient.createContainerCmd("busybox").withCmd("top")
+        CreateContainerResponse container = dockerClient.createContainerCmd("busybox").withUser("root").withCmd("top")
```

Comment:
Fair enough. I guess my thinking was that setting the user, even though it's the default, would show that it works as expected without having to create a new test for this one option.